python/pyrogue/utilities/fileio/_FileReader.py

```diff
@@ -92,7 +92,7 @@ def _nextRecord(self):
                 try:
                     self._data = numpy.fromfile(self._currFile, dtype=numpy.int8, count=payload)
                 except Exception:
-                    raise rogue.GeneralError(f"fileio.FileReader","Failed to read data from {self._currFname}")
+                    raise rogue.GeneralError("fileio.FileReader","Failed to read data from {self._currFname}")
```

Comment:
Doesn't `"Failed to read data from {self._currFname}"` need to be an f-string.

Reply:
https://github.com/slaclab/rogue/pull/660/commits/0ff511a4795626a42074487fe954adbdbda7d936
Fixed in the travis-focal branch